.github/workflows/package-and-test.yml

```diff
@@ -0,0 +1,21 @@
+name: (Maven) package and test custom Kafka Connect Connector
+
+on: [pull_request]
+
+jobs:
+  package-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'zulu'
+      - name: Install upstream dependencies
+        run: ./install_upstream_dependencies.sh
+      - name: Package up ZIP
+        run: mvn --batch-mode package -DskipTests
+      - name: Run UdxStreamPartitioner unit tests
+        run: mvn --batch-mode -Dtest=UdxStreamPartitionerTest -Dsurefire.failIfNoSpecifiedTests=false clean test
```

Comment:
I think we could also add a deploy step to upload the latest built .zip to the S3 bucket (a similar pattern to what we already do with docker images and uploading new ones to ECR

Reply:
All done ✅